tests/test-extensions/petstore-extension/src/main/java/org/mule/test/petstore/extension/PetStoreConnector.java

```diff
@@ -64,6 +64,9 @@
   @Optional
   private Aquarium aquarium;
 
+  @ParameterGroup(name = "Advanced Leash Configuration", showInDsl = true)
+  public AdvancedLeashConfiguration advancedLeashConfiguration;
```

Comment:
add a comment to make it clear that the odd thing about this are the spaces in the name

Reply:
Done